plotheatmap/plotheatmap.xml

```diff
@@ -0,0 +1,103 @@
+<tool id="plotheatmap" name="plotHeatmap" version="1.0">
+    <description>Plot heatmap</description>
+    <requirements>
+        <requirement type="package" version="3.0.1">r-gplots</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command>
```

Comment:
You can remove the `<stdio>` section above and use:
```xml
<command detect_errors="aggressive">
```
which will also look for the 'error:' string in stderr, see https://docs.galaxyproject.org/en/latest/dev/schema.html#detect-errors